content/en/data_security/guide/tls_ciphers_deprecation.md

```diff
@@ -0,0 +1,62 @@
+
+## Disabled Cipher Suites
+
+| Code       | IANA Name                                |
+|------------|------------------------------------------|
+| 0xC0,0x27   | TLS_ECDHE_RSA_WITH_AES_128_CBC_SHA256    |
+| 0xC0,0x23   | TLS_ECDHE_ECDSA_WITH_AES_128_CBC_SHA256  |
+| 0xC0,0x28   | TLS_ECDHE_RSA_WITH_AES_256_CBC_SHA384    |
+| 0xC0,0x24   | TLS_ECDHE_ECDSA_WITH_AES_256_CBC_SHA384  |
+| 0x00,0x3C   | TLS_RSA_WITH_AES_128_CBC_SHA256          |
+| 0x00,0x3D   | TLS_RSA_WITH_AES_256_CBC_SHA256          |
+
+After this date, the following cipher suites will be accepted:
+
+## Accepted Cipher Suites
+
+| Code       | IANA Name                                       |
+|------------|-------------------------------------------------|
+| 0xC0,0x2B   | TLS_ECDHE_ECDSA_WITH_AES_128_GCM_SHA256        |
+| 0xC0,0x2F   | TLS_ECDHE_RSA_WITH_AES_128_GCM_SHA256          |
+| 0xC0,0x2C   | TLS_ECDHE_ECDSA_WITH_AES_256_GCM_SHA384        |
+| 0xC0,0x30   | TLS_ECDHE_RSA_WITH_AES_256_GCM_SHA384          |
+| 0xCC,0xA9   | TLS_ECDHE_ECDSA_WITH_CHACHA20_POLY1305_SHA256 |
+| 0xCC,0xA8   | TLS_ECDHE_RSA_WITH_CHACHA20_POLY1305_SHA256   |
+| 0xC0,0x09   | TLS_ECDHE_ECDSA_WITH_AES_128_CBC_SHA          |
+| 0xC0,0x13   | TLS_ECDHE_RSA_WITH_AES_128_CBC_SHA            |
+| 0xC0,0x0A   | TLS_ECDHE_ECDSA_WITH_AES_256_CBC_SHA          |
+| 0xC0,0x14   | TLS_ECDHE_RSA_WITH_AES_256_CBC_SHA            |
+| 0x00,0x9C   | TLS_RSA_WITH_AES_128_GCM_SHA256               |
+| 0x00,0x9D   | TLS_RSA_WITH_AES_256_GCM_SHA384               |
+| 0x00,0x2F   | TLS_RSA_WITH_AES_128_CBC_SHA                 |
+| 0x00,0x35   | TLS_RSA_WITH_AES_256_CBC_SHA                 |
+| 0x13,0x01   | TLS_AES_128_GCM_SHA256                        |
+| 0x13,0x02   | TLS_AES_256_GCM_SHA384                        |
+| 0x13,0x03   | TLS_CHACHA20_POLY1305_SHA256                  |
+
+This change does not impact Datadog for Government site for which the accepted cipher suites remain the following:
+
+## Accepted Cipher Suites (Datadog for Government)
+
+| Code       | IANA Name                                |
+|------------|------------------------------------------|
+| 0xC0,0x2F   | TLS_ECDHE_RSA_WITH_AES_128_GCM_SHA256    |
+| 0xC0,0x30   | TLS_ECDHE_RSA_WITH_AES_256_GCM_SHA384    |
+| 0xC0,0x2B   | TLS_ECDHE_ECDSA_WITH_AES_128_GCM_SHA256  |
+| 0xC0,0x2C   | TLS_ECDHE_ECDSA_WITH_AES_256_GCM_SHA384  |
+
+# Client Compatibility
+
+Datadog's systems already require the use of **TLS 1.2** and compatible clients will be able to negotiate other cipher suites. However, specific client-side configurations may alter this behavior. Use the client of your choice to connect to [tls-config-test.datadoghq.com][3] which is configured with the target ciphers, or use [How's my SSL? API][1] to check the cipher suites it supports. For any additional questions, reach out to [Datadog support][2].
```

Comment:
Since we mention contacting support, if the author hasn't already, they need to add an entry to this Confluence doc so that support knows what to do: https://datadoghq.atlassian.net/wiki/spaces/TS/pages/328404964/The+Public+Docs+say+Contact+Support

Reply:
Thanks @maycmlee great catch! @simnyc would you be able to add some information to this Confluence page regarding this issue, before we publish/merge this ?

Reply:
done ! https://datadoghq.atlassian.net/wiki/spaces/TS/pages/328404964/The+Public+Docs+say+Contact+Support#TLS-Cipher-Suites-Changes